{"uid":"190a6bd38d8c7db3","name":"test_network_cut","fullName":"tests.integration.ha_tests.test_ha#test_network_cut","historyId":"1cdeb10b807ab13ac47904465f0bc613","time":{"start":1729476421726,"stop":1729476421726,"duration":0},"status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1729474435966,"stop":1729474435966,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1729474435966,"stop":1729474435966,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1729474435967,"stop":1729474436266,"duration":299},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"chaos_mesh","time":{"start":1729476421727,"stop":1729476443149,"duration":21422},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1729476443149,"stop":1729476443149,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"676391d8e9722632","name":"log","source":"676391d8e9722632.txt","type":"text/plain","size":5744}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"chaos_mesh::0","time":{"start":1729476443151,"stop":1729476450372,"duration":7221},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1729476450372,"stop":1729476450994,"duration":622},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1729476450995,"stop":1729476450995,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1729476450995,"stop":1729476450995,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1729476450995,"stop":1729476450995,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_ha"},{"name":"host","value":"fv-az2021-977"},{"name":"thread","value":"13748-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_ha"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"e167d169d7b59e8f","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":11,"skipped":1,"passed":56,"unknown":1,"total":69},"items":[{"uid":"4ed7116a5e9ed3d8","reportUrl":"../2693//#testresult/4ed7116a5e9ed3d8","status":"passed","time":{"start":1729389583584,"stop":1729389818586,"duration":235002}},{"uid":"479eefb7157405a7","reportUrl":"../2692//#testresult/479eefb7157405a7","status":"passed","time":{"start":1729303335116,"stop":1729303639191,"duration":304075}},{"uid":"f85dc50b9a32bd8","reportUrl":"../2666//#testresult/f85dc50b9a32bd8","status":"passed","time":{"start":1729217050744,"stop":1729217283798,"duration":233054}},{"uid":"1cea067377ba2936","reportUrl":"../2634//#testresult/1cea067377ba2936","status":"passed","time":{"start":1729130567827,"stop":1729130796905,"duration":229078}},{"uid":"2440eb380ede635b","reportUrl":"../2616//#testresult/2440eb380ede635b","status":"passed","time":{"start":1729043931868,"stop":1729044226678,"duration":294810}},{"uid":"2b077877eff36714","reportUrl":"../2589//#testresult/2b077877eff36714","status":"passed","time":{"start":1728957567264,"stop":1728957861944,"duration":294680}},{"uid":"ad468198ee48cd72","reportUrl":"../2570//#testresult/ad468198ee48cd72","status":"passed","time":{"start":1728871175962,"stop":1728871404982,"duration":229020}},{"uid":"cb10901ac3a094ea","reportUrl":"../2567//#testresult/cb10901ac3a094ea","status":"passed","time":{"start":1728784726479,"stop":1728785021115,"duration":294636}},{"uid":"adae471a05782475","reportUrl":"../2566//#testresult/adae471a05782475","status":"passed","time":{"start":1728698375791,"stop":1728698605900,"duration":230109}},{"uid":"f34b3a9a586a3396","reportUrl":"../2536//#testresult/f34b3a9a586a3396","status":"passed","time":{"start":1728612038056,"stop":1728612261655,"duration":223599}},{"uid":"9928a28bc3bbd4a7","reportUrl":"../2533//#testresult/9928a28bc3bbd4a7","status":"passed","time":{"start":1728526890749,"stop":1728527121155,"duration":230406}},{"uid":"dca4af4d63e7938f","reportUrl":"../2514//#testresult/dca4af4d63e7938f","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-0/charm/\"","time":{"start":1728439225361,"stop":1728439309228,"duration":83867}},{"uid":"f2d9bdb41d79592b","reportUrl":"../2493//#testresult/f2d9bdb41d79592b","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-0/charm/\"","time":{"start":1728352712121,"stop":1728352795894,"duration":83773}},{"uid":"daa35ef001cd6094","reportUrl":"../2484//#testresult/daa35ef001cd6094","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-1/charm/\"","time":{"start":1728267390586,"stop":1728267575731,"duration":185145}},{"uid":"caccf62ab90a3c28","reportUrl":"../2483//#testresult/caccf62ab90a3c28","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-0/charm/\"","time":{"start":1728179889077,"stop":1728179972314,"duration":83237}},{"uid":"c33cd1d3e952f770","reportUrl":"../2481//#testresult/c33cd1d3e952f770","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-2/charm/\"","time":{"start":1728007143299,"stop":1728007318605,"duration":175306}},{"uid":"59295723e6d49ff4","reportUrl":"../2477//#testresult/59295723e6d49ff4","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-2/charm/\"","time":{"start":1727922476803,"stop":1727922659291,"duration":182488}},{"uid":"89f83511cd05e1d9","reportUrl":"../2465//#testresult/89f83511cd05e1d9","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-1/charm/\"","time":{"start":1727834567058,"stop":1727834649661,"duration":82603}},{"uid":"63b5afeacac38df4","reportUrl":"../2456//#testresult/63b5afeacac38df4","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-2/charm/\"","time":{"start":1727747905465,"stop":1727748089832,"duration":184367}},{"uid":"f96bf8a134fe1dd9","reportUrl":"../2444//#testresult/f96bf8a134fe1dd9","status":"broken","statusDetails":"pymongo.errors.InvalidURI: Bad database name \"var/lib/juju/agents/unit-mongodb-k8s-1/charm/\"","time":{"start":1727662638205,"stop":1727662740620,"duration":102415}}]},"tags":["asyncio","group(1)"]},"source":"190a6bd38d8c7db3.json","parameterValues":[]}